src/frame/mod.rs

```diff
@@ -20,6 +20,12 @@ pub use plane::*;
 
 const FRAME_MARGIN: usize = 16 + SUBPEL_FILTER_SIZE;
 
+/// Optional per-frame encoder parameters
+pub struct FrameParameters {
+  /// Force the frame to be emitted as keyframe
```

Comment:
Is this sufficient (the bool)?

In the future we may want to force frames to be switch frames or intra-only frames.

Reply:
A frame-type enum makes sense to me (for now it will only have one entry).

Reply:
Done.